run.py

```diff
@@ -21,10 +18,12 @@
 from cogs.voice import Voice
 from discord_bot import HuskieBot
 
+
 def setup_bot():
-    bot = HuskieBot(
+    bot_ = HuskieBot(
```

Comment:
Why rename it to `bot_`? within the context of this function, there's only this `bot` and this `bot` won't exist outside of the function besides it's reference getting passed out via the return.

Reply:
It is not a PEP standard as far as I know, but it best practice to not name a variable the same as an outer scope variable. A PEP8 standard to name something "the same" is to append an underscore to denote the difference. Pycharm calls it shadowing.